### DISCUSSION TITLE
Restream idea can add referer

### DISCUSSION THREAD

**Discussion by crewockeez** [datarhei/restreamer#618] [Ideas]
i restream link intent:https://live.vip-streaming.com:30443/cloudstreaming/true-spotv-1/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9MTEvMjMvMjAyMiA5OjI4OjI3IEFNJmhhc2hfdmFsdWU9bFVSR3JPenlqOUVwQno1WE1BM2ROZz09JnZhbGlkbWludXRlcz0yMA==#Intent;package=com.wiseplay;S.title=DOO-TV^^;S.referer=https://ufabetcompany.com/;end

it use referer

**Comment by mrdastt** [comment]
Hi @crewockeez, I actually just sent in a PR to add this feature. You can see the request here:

https://github.com/datarhei/restreamer-ui/pull/40

I can also send you the docker hub link if it's not merged soon
